.tools/readmes/config.py

```diff
@@ -44,7 +45,6 @@
             "service_folder": 'javav2/example_code/{{service["name"]}}',
             "sdk_api_ref": 'https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/services/{{service["name"]}}/package-summary.html',
             "service_folder_overrides": {
-                "sesv2": "javav2/example_code/ses",
```

Comment:
In Java, SESv2 and SES use the same folder - but that's not quite what service_folder_overrides does. Please cut a 2.5 for Scott to fix this.

Reply:
Good point, I created #6388 to track this.